pkg/sql/crdb_internal.go

```diff
@@ -5651,20 +5651,21 @@ GROUP BY
 
 var crdbInternalActiveRangeFeedsTable = virtualSchemaTable{
 	comment: `node-level table listing all currently running range feeds`,
-	// NB: startTS is exclusive; consider renaming to startAfter.
 	schema: `
 CREATE TABLE crdb_internal.active_range_feeds (
   id INT,
   tags STRING,
-  startTS STRING,
```

Comment:
renaming a column isn't great for a backport, maybe you want to walk this one back if you're considering backporting this. Adding columns - maybe, hopefully, ok.

Reply:
Ack. Going to keep old name.